sr_grasp/scripts/quick_grasp

```diff
@@ -59,6 +60,15 @@ class QuickGraspNode(object):
         else:
             rospy.logerr("Fail!")
 
+    def new_grasp(self, name):
+        grasp = Grasp()
+        grasp.id = name
+        positions = self.hand.read_all_current_positions()
+        for k, v in positions.iteritems():
+            positions[k] = radians(v)
```

Comment:
so we store them in degrees in the yaml file (same as our old xml file). Probably ok as it's easier to read it, but wondering if it'd be cleaner to just have everything in radians directly. Thoughts?